api/envoy/api/v2/eds.proto

```diff
@@ -80,6 +81,11 @@ message ClusterLoadAssignment {
     //    "lb"_drop = 20%  // 50% of the remaining 'actual' load, which is 40%.
     //    actual_outgoing_load = 20% // remaining after applying all categories.
     repeated DropOverload drop_overloads = 2;
+
+    // Priority levels and localities are considered overprovisioned with this factor. This means
```

Comment:
I don't think I understand what the value means. From reading this I assumed that if over_provisioning_factor was 2, then a locality would be considered healthy until half its hosts were unhealthy but the default is 140 which doesn't make sense. Maybe make it clearer that this is intended to be a percentage, and probably include one or more examples.